linux.sh

```diff
@@ -16,42 +16,6 @@
 # specific language governing permissions and limitations
 # under the License.
 verify_gcc_enable(){
-  feature="$1"
-  if [ "$feature" = "BUSTACHE_ENABLED" ]; then
-    if (( COMPILER_MAJOR == 6 && COMPILER_MINOR >= 3 && COMPILER_REVISION >= 1  )); then
-      echo "true"
-    elif (( COMPILER_MAJOR > 6 )); then
-      echo "true"
-    else
-      echo "false"
-    fi
-  elif [ "$feature" = "EXECUTE_SCRIPT_ENABLED" ]; then
-    if (( COMPILER_MAJOR >= 6 )); then
-      echo "true"
-    else
-      echo "false"
-    fi
-  elif [ "$feature" = "KAFKA_ENABLED" ]; then
-    if (( COMPILER_MAJOR >= 4 )); then
-      if (( COMPILER_MAJOR > 4 || COMPILER_MINOR >= 8 )); then
-        echo "true"
-      else
-        echo "false"
-      fi
-    else
-      echo "false"
-    fi
-   elif [ "$feature" = "PCAP_ENABLED" ]; then
-    if (( COMPILER_MAJOR >= 4 )); then
-      if (( COMPILER_MAJOR > 4 || COMPILER_MINOR >= 8 )); then
-        echo "true"
-      else
-        echo "false"
-      fi
-    else
-      echo "false"
-    fi
-  else
-    echo "true"
-  fi
+  #feature="$1"
```

Comment:
can this line be removed?

Reply:
My intention was to leave it there as a documentation for future extension of this function, because it might not be obvious that the first argument is the feature name. Currently all the features are working on all of the supported compilers, due to dropping old ones, but if this ever changes, here is the place to put those checks.

Reply:
Makes sense, thanks!